scripts/dev/ci/travis_run.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 if [[ $DOCKER ]]; then
-    docker run --privileged -v $PWD:/outside thecompiler/qutebrowser:$DOCKER
+    docker build -t img misc/docker/$DOCKER
+    docker run --privileged -v $PWD:/outside img
```

Comment:
I added the dependencies to the `master` branch in 0b7a71e18eea7a0018fdaca67ac2be1bf47f6310 - can you please remove this change and the changes to the `Dockerfile`s from your PR again?